src/core/nginx.go

```diff
@@ -399,6 +399,10 @@ func (n *NginxBinaryType) WriteConfig(config *proto.NginxConfig) (*sdk.ConfigApp
 		if _, ok = fileDeleted[file]; ok {
 			continue
 		}
+		if found, _ := FileExists(file); !found {
+			log.Debugf("skip delete for non-existing file: %s", file)
```

Comment:
Might there be any information in the error returned by `FileExists` that would be potentially useful to have in the log?

Reply:
sure, will change the log

Reply:
added error check